### ISSUE TITLE
Unable to forecast negative values

### ISSUE THREAD

**Issue by fxnitsch** [jdb78/pytorch-forecasting#560]
- PyTorch-Forecasting version: 0.9.0
- PyTorch version: 1.8.1
- Python version: 3.8.10
- Operating System: Win10

_Thank you Jan for your impressive work! I really enjoy working and experimenting with your package 💪_ 

### Expected behavior

My training data consists of around 50,000 timesteps of which around 2% are negative `target` values (the others range from 0 to around 200).
After training, I was able to achieve overall good results, e.g.
![best_prediction_22](https://user-images.githubusercontent.com/29756334/121482814-c5dce980-c9cd-11eb-8c2e-dc9b4cb3de4c.png)

Since I also have negative `target` values in my training data set, I expected my model to be also applicable for forecasting negative `target` values.

### Actual behavior

However, when looking at the sample results based on the highest `SMAPE`, I found that my model fails at forecasting negative `target` values at all. Please find a few examples:
![worst_prediction_1](https://user-images.githubusercontent.com/29756334/121483457-81058280-c9ce-11eb-8e30-c8f24252f1cc.png)
![worst_prediction_3](https://user-images.githubusercontent.com/29756334/121483459-819e1900-c9ce-11eb-9e54-37c187fbb14a.png)
![worst_prediction_7](https://user-images.githubusercontent.com/29756334/121483460-8236af80-c9ce-11eb-8b74-97f8163328eb.png)

Maybe this has to do with `scaling` or `grouping`? Do you have any ideas on which parameter to check? Any help is much appreciated.

### Code to reproduce the problem
Unfortunately, I can not share my data with you. However, please find my model parameterization below.
```python
training = TimeSeriesDataSet(
        data[lambda x: x.time_idx <= training_cutoff],
        time_idx="time_idx",
        target="DE_target",
        group_ids=["country_ISO"],
        min_encoder_length=24,  
        max_encoder_length=24,
        min_prediction_length=6,
        max_prediction_length=6,
        static_categoricals=["country_ISO"],
        time_varying_known_categoricals=["workday", "holiday", "dayofweek", "hourofday"],
        time_varying_known_reals=["time_idx",
                                  "DE_load_forecast_entsoe_transparency",
                                  "DE_solar_generation_actual",
                                  "DE_wind_offshore_generation_actual",
                                  "DE_wind_onshore_generation_actual",],
        time_varying_unknown_categoricals=[],
        time_varying_unknown_reals=["DE_target", "DE_load_actual_entsoe_transparency"],
        target_normalizer=GroupNormalizer(groups=["country_ISO"], transformation="softplus"),  
        add_relative_time_idx=True,
        add_target_scales=True,
        add_encoder_length=True,
        allow_missings=False
    )

    tft = TemporalFusionTransformer.from_dataset(
        training,
        learning_rate=0.00535274,
        hidden_size=121,  
        attention_head_size=1,
        dropout=0.180093,  
        hidden_continuous_size=37,  
        output_size=7,  
        loss=QuantileLoss(),
        reduce_on_plateau_patience=4,
    )
```

**Comment by jdb78** [comment]
This line is your issue `target_normalizer=GroupNormalizer(groups=["country_ISO"], transformation="softplus"),`

`"softplus"` means to allow only positive values. See the [documention](https://pytorch-forecasting.readthedocs.io/en/latest/api/pytorch_forecasting.data.encoders.GroupNormalizer.html) for your options.

**Comment by fxnitsch** [comment]
I have missed this setting completely, thank you for pointing that out so quickly!

**Comment by cjing1** [comment]
> I have missed this setting completely, thank you for pointing that out so quickly!

May I ask what transformation did you end up using?

**Comment by fxnitsch** [comment]
I made a simple normalization on my series in preprocessing.

**Comment by cjing1** [comment]
Thank you for the information.
